kernel/src/main/java/org/kframework/krun/KRunOptions.java

```diff
@@ -217,36 +216,30 @@ public SearchType searchType() {
     @Parameter(names="--depth", description="The maximum number of computational steps to execute or search the definition for.")
     public Integer depth;
 
-    @ParametersDelegate
-    public Experimental experimental = new Experimental();
-
-    public final class Experimental {
-
-        @Parameter(names="--statistics", description="Print rewrite engine statistics.", arity=1,
-                converter=OnOffConverter.class)
-        public boolean statistics = false;
+    @Parameter(names="--statistics", description="Print rewrite engine statistics.", arity=1,
+            converter=OnOffConverter.class)
+    public boolean statistics = false;
 
-        @Parameter(names="--debugger", description="Run an execution in debug mode.")
-        public boolean debugger = false;
+    @Parameter(names="--debugger", description="Run an execution in debug mode.")
+    public boolean debugger = false;
 
-        @Parameter(names="--ltlmc", description="Specify the formula for model checking at the commandline.")
-        public String ltlmc;
+    @Parameter(names="--ltlmc", description="Specify the formula for model checking at the commandline.")
```

Comment:
this should definitely be removed because it hasn't worked since the maude backend.

Reply:
Removed.